src/Dispatcher.cpp

```diff
@@ -0,0 +1,24 @@
+#include "Dispatcher.hpp"
+#include "ILogEntry.hpp"
+#include <cstddef>
+#include <iostream>
+
+void Dispatcher::registerInformation(const ILogEntry& log)
+{
+    const InformationData data{log.getDate(), log.getServerName(), log.getName(), log.getMessage()};
+    m_values.push_back(data);
+}
+
+void Dispatcher::dispatchInformation()
+{
+    for (std::size_t i = m_disptachCount; i < m_values.size(); i++)
```

Comment:
if you would have used queue instead of vector

you could just do 

```
while(not queue.empty())
{
dispach(queue.pop); // dispatch might be just private function whcih calls cout  - for now :)
}
```